File: arry_ele_freq_sec_way_31.pl
#!/usr/bin/perl

### Array element frequency ###
use Data::Dumper;
print("Enter a string: ");
$str=<STDIN>;
chomp ($str);
@array=split(/\s+/,$str);
%hash;
foreach $word (@array){
	$hash{$word}++;
}
foreach $ele (keys %hash){
	push @num, $hash{$ele};
	print("$ele:->$hash{$ele}\n");
}
#print(Dumper \@num);

File: arry_ele_freq_sec_way_31.py
#!/usr/bin/env python3
# Array element frequency
from collections import Counter

user_input = input("Enter a string: ")
words = user_input.split()
word_freq = Counter(words)

num = []
for word, count in word_freq.items():
	num.append(count)
	print(f"{word}:->{count}")

print(num)
